labs/src/main/scala/org/scalalabs/basic/lab02/CollectionExercise.scala

```diff
@@ -76,6 +100,7 @@ object CollectionExercise04 {
    * To keep it simple it's ok to use String.split to extract all words of a sentence.
    */
   def calcLengthLongestWord(lines: String*): Int = {
-    error("fix me")
+    val words = lines.map(s => s.split(" ")).flatten
```

Comment:
I completely missed the `flatten` function and its relatives until reading the sample solutions. Good job for noticing and using it!